File: app/menu/menu.go
package menu

import (
	"io/ioutil"
	"encoding/json"
	"path/filepath"
	"github.com/revel/revel"
)

type MenuItems struct {
	Category []struct {
		Items []struct {
			Description string  `json:"description"`
			Name        string  `json:"name"`
			Price       float64 `json:"price"`
		} `json:"items"`
		Name string `json:"name"`
	} `json:"Category"`
}

var Menu MenuItems

func Load() {
	menuJSON, err := ioutil.ReadFile(filepath.Join(revel.AppPath, "menu/menu.json"))
	if err != nil {
		revel.ERROR.Println("failed to read menu json file:", err)
	}

	err = json.Unmarshal(menuJSON, &Menu)
	if err != nil {
		revel.ERROR.Println("failed to read menu json data:", err)
	}
}

File: app/menu/menu.js
var menu = {};

$(".boxwhite").each(function(i) {
	let name = $(this).find(".dishhead").text();
	if (name == "")
		return;
	name = name.substr(0, name.length - 2);
	let description = $(this).find(".dishcont").text();
	let price = $(this).find(".dishpricelist").text();
	try {
		menuPrice = parseFloat(menuPrice.match(/(?:\$)(\d+\.\d+)/)[1]);
	} catch (err) { }
	
	let category;
	for (i = 0; i < 100; ++i) {
		let current = $(this).prev();
		for (j = 0; j < i; j++) {
			current = current.prev();//.text();
        }
		if (current.hasClass("futitle")) {
			category = current.text();
			if (category)
				break;
        }
    }
	if (!menu[category])
		menu[category] = [];
	
	menu[category].push({
		"name": name,
		"description": description,
		"price": price
    });
});
console.log(JSON.stringify(menu));